contracts/curve/USDThreePoolValueTest.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "../AnteTest.sol";
+import "../interfaces/IERC20.sol";
+
+/// @title Curve 3Pool Balance Test
+/// @notice Ensure that at least 2 of three tokens are balanced within 25% of each other
+/// @dev At different times, there will be different trading volume for each pair
+/// based on interest. For example, 1 year ago, USDT was balanced within 15% of the others
+/// and now it only makes up half compared to another token in the pool.
+/// Based on three time points; today, 6 months ago, and 1 year ago, the highest variation was
+/// 20%. Based on this, the threshold will be a 25% difference.
+/// Eg (100, 120, 60) will pass and (100, 150, 60) will fail
```

Comment:
The way `isSafe` is written, it doesn't check for the 25% balanced within each other. But it does test if the total balances are above 10M. If that's the intent, let's clarify these comments to show that!